ios/brave-ios/Sources/BraveWallet/Crypto/WalletActions/DepositTokenView.swift

```diff
@@ -265,22 +277,32 @@ private struct DepositDetailsView: View {
           .fontWeight(.semibold)
           .foregroundColor(Color(.bravePrimary))
         MultipleNetworkIconsView(networks: networks.filter({ $0.coin == .fil }))
-      case .zec, .btc:
+      case .btc:
+        Text(Strings.Wallet.btcAccountDescription)
+          .fontWeight(.semibold)
+          .foregroundColor(Color(.bravePrimary))
+        MultipleNetworkIconsView(networks: networks.filter({ $0.coin == .btc }))
+      case .zec:
         EmptyView()
       @unknown default:
         EmptyView()
       }
     }
   }
 
-  @ViewBuilder private func qrCodeView(_ account: BraveWallet.AccountInfo) -> some View {
+  struct QRCodeViewModel {
```

Comment:
Should be `private` too. I don't _think_ we display QR codes anywhere else but Deposit after https://github.com/brave/brave-browser/issues/37029 so fine to keep in this file.